frontend-react/src/utils/CustomRenderUtils.tsx

```diff
@@ -50,6 +51,25 @@ const SessionWrapper =
         );
     };
 
+export const QueryWrapper =
+    (client: QueryClient = new QueryClient()) =>
+    ({ children }: PropsWithChildren<{}>) =>
+        <QueryClientProvider client={client}>{children}</QueryClientProvider>;
+
+const AppWrapper =
```

Comment:
realizing now that this wrapper i created is unused. But maybe we should be using it everywhere by default?

Reply:
It looks like you're using this in `ValueSetsIndex` tests via `renderWithFullAppContext`?

Reply:
o yeah, tru